code/game/atoms_movable.dm

```diff
@@ -116,6 +120,34 @@
 		return 1
 	return 0
 
+/mob/living/forceMove(atom/destination)
+	stop_pulling()
+	if(pulledby)
+		pulledby.stop_pulling()
+	if(buckled)
+		buckled.unbuckle_mob()
+	if(buckled_mob)
+		unbuckle_mob(force=1)
+	. = ..()
+	if(client)
+		reset_perspective(destination)
+	if(pipes_shown.len && !istype(loc, /obj/machinery/atmospherics))
+		remove_ventcrawl()

```

Comment:
We add pulling and buckle stops to mob's forceMove() instead of copypasting them after each forceMove() call.